apps/site/assets/ts/hooks/useVehiclesChannel.ts

```diff
@@ -0,0 +1,137 @@
+import { pick, keyBy } from "lodash";
+import deepEqual from "fast-deep-equal/react";
+import { Reducer } from "react";
+import { DirectionId } from "../__v3api";
+import useChannel from "./useChannel";
+import { CrowdingType } from "../schedule/components/__schedule";
+
+/**
+ * The format of a vehicle emitted by the VehiclesChannel
+ */
+export interface VehicleData {
```

Comment:
This looks like `VehicleData` has the exact same properties as `Vehicle`.  Is this still needed?

Reply:
True, I suppose it is not needed just yet! I suspect we'll be adding a last updated timestamp to the Vehicle so there will be some parsing added soon here. I'll have `Vehicle` extend `VehicleData` so it is at least less redundant for now! 

I also am in the habit of doing a `XData => X` conversion to change snake case fields to camelCase to work with in javascript, but it seems dotcom's convention is to keep the snake case fields so I'll stick with that convention!